Bottom: 5d2a1108c4b025e853b91ee7c4a5e7b53d0f9f73
Top:    d3891eb81adee57b5e323b157b1059e5585d6222
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-15 12:32:33 +0100

Refresh of use-trackball-to-allow-rotatio

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 53bb33b..7d5f6ef 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1707,6 +1707,7 @@ ghid_flip_view (Coord center_x, Coord center_y, bool flip_x, bool flip_y)
   ghid_invalidate_all ();
 }
 
+void
 ghid_view_2d (void *ball, gboolean view_2d, gpointer userdata)
 {
   global_view_2d = view_2d;
